mcp-server/docs/guides/DEBUG-PAT.md

```
@@ -0,0 +1,128 @@
+### 1. PAT Not Set
+
+```
+[Auth] ⚠️  No authentication method configured!
+[Auth]   personalAccessToken: not configured
+```
+
+**Solution:** Check that PAT is provided via MCP client configuration (query parameters or headers), not Docker environment variables.
+
+### 2. Incorrect Header Format
+
+If header doesn't start with `Token `, check:
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)